doc/index.rst

```diff
@@ -41,7 +41,7 @@ Please only do this if you are sure it will not interfere with operations:
 
```

Comment:
This is about line 34. I kindly suggest to replace `When MTMount is started` with `When the MTMount CSC is started`.